js/build_map.js

```diff
@@ -1,14 +1,43 @@
-// Width and height
-var chart_width     =   1000;
-var chart_height    =   700;
-
-// define categories
+// Define one big global to eventually rule them all.
+// null values are placeholders for globals that will be filled in
+var waterUseViz = {
+  dims: {
+    map: {
+      width: 1000,
+      height: 700
+    },
+    buttonBox: {
+      widthDesktop: 250,
+      heightDesktop: 250,
+      width: null,
+      height: null
+    },
+    svg: {
+      width: null,
+      height: null
+    },
+    watermark: {
+      width: null,
+      height: null
+    }
+  },
+  elements: {
```

Comment:
What is the role of these entries?

Reply:
i picture this eventually containing all of the DOM elements that we want to keep as globals - `svg` and `map` for sure, probably `watermark` and `mapBackground`. For now I'm just adding the one DOM element that I'm currently editing (`buttonBox`) so I don't change too much code at once.